src/test/java/io/strimzi/kafka/bridge/http/base/HttpBridgeTestBase.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.http.base;
+
+import io.strimzi.kafka.bridge.HealthChecker;
+import io.strimzi.kafka.bridge.clients.BasicKafkaClient;
+import io.strimzi.kafka.bridge.config.BridgeConfig;
+import io.strimzi.kafka.bridge.config.KafkaConfig;
+import io.strimzi.kafka.bridge.config.KafkaConsumerConfig;
+import io.strimzi.kafka.bridge.config.KafkaProducerConfig;
+import io.strimzi.kafka.bridge.facades.AdminClientFacade;
+import io.strimzi.kafka.bridge.http.HttpBridge;
+import io.strimzi.kafka.bridge.http.HttpConfig;
+import io.strimzi.kafka.bridge.http.services.BaseService;
+import io.strimzi.kafka.bridge.http.services.ConsumerService;
+import io.strimzi.kafka.bridge.http.services.ProducerService;
+import io.strimzi.kafka.bridge.http.services.SeekService;
+import io.strimzi.kafka.bridge.utils.Urls;
+import io.vertx.core.Vertx;
+import io.vertx.ext.web.client.WebClient;
+import io.vertx.ext.web.client.WebClientOptions;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testcontainers.containers.KafkaContainer;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+import static io.strimzi.kafka.bridge.Constants.HTTP_BRIDGE;
+
+@ExtendWith(VertxExtension.class)
+@SuppressWarnings({"checkstyle:JavaNCSS"})
+@Tag(HTTP_BRIDGE)
+public abstract class HttpBridgeTestBase {
+
+    protected static final Logger LOGGER = LogManager.getLogger(HttpBridgeTestBase.class);
+    protected static Map<String, Object> config = new HashMap<>();
+
+    // for periodic/multiple messages test
+    protected static final int PERIODIC_MAX_MESSAGE = 10;
+    protected static final int PERIODIC_DELAY = 200;
+    protected static final int MULTIPLE_MAX_MESSAGE = 10;
+    protected static final int TEST_TIMEOUT = 60;
+    protected int count;
+
+    public static final KafkaContainer KAFKA_CONTAINER;
```

Comment:
Why naming a variable/member like a constant?

Reply:
Hmmm, for instance, we using KafkaContainer and it is semantically best name for this. Do you have any alternatives? Maybe, `KAFKA` or `STRIMZI_KAFKA` would better fit in this? @ppatierno